packages/shared/lib/services/account.service.ts

```diff
@@ -111,4 +112,84 @@ class AccountService {
     }
 }
 
+function parseTeamName({ name, email }: { name: string; email?: string | undefined }): string {
```

Comment:
I guess it could be emailToTeamName and only return false | string to avoid all the repetitions